packages/puppeteer-core/src/node/ScreenRecorder.ts

```diff
@@ -165,7 +188,10 @@ export class ScreenRecorder extends PassThrough {
         // Filters to ensure the images are piped correctly,
         // combined with any format-specific filters.
         ['-vf', filters.join()],
-        'pipe:1',
+        // Write to provided path, else pipe to stdout.
+        path ?? 'pipe:1',
+        // Overwrite output, or exit immediately if file already exists.
+        [overwrite ? '-y' : '-n'],
```

Comment:
> Why don't we need overwrite (`-y`)?

@Lightning00Blade I added it back here, behind an `overwrite` option (`true` by default, so no API change…)